.changeset/witty-crabs-change.md

```diff
@@ -0,0 +1,5 @@
+---
+"@coinbase/onchainkit": patch
+---
+
+**feat**: WalletDropdownFundLink by @0xAlec #1021
```

Comment:
One more thing we need to learn to be good with our Changelog, is to add more readable informations.

Put yourself in the shoes of someone reading this. There is not much meaning in this sentence.